llvm/docs/ReleaseNotes.md

```diff
@@ -350,10 +350,16 @@ Changes to the Debug Info
 Changes to the LLVM tools
 ---------------------------------
 
+* llvm-objcopy now supports the following options for MachO:
+  `--globalize-symbol`, `--globalize-symbols`,
+  `--keep-global-symbol`, `-G`, `--keep-global-symbols`,
+  `--localize-symbol`, `-L`, `--localize-symbols`,
+  `--skip-symbol`, `--skip-symbols`.
+
 Changes to LLDB
 ---------------------------------
 
-* LLDB now now supports inline diagnostics for the expression evaluator and command line parser.
+* LLDB now supports inline diagnostics for the expression evaluator and command line parser.
```

Comment:
Related to how cherry-picks work, I'd put this in its own separate commit that you push directly, since it's completely unrelated.

Reply:
Reverted in a fixup. I'll push a separate commit to main directly.